forms/views/admin/create.php

```diff
@@ -8,7 +9,6 @@
 
 
 error_reporting(E_ALL);
-ini_set('display_errors', 1);

```

Comment:
Remove this line so we are not displaying error directly

Reply:
You likely want to remove `error_reporting(E_ALL);` as well